README.md

```diff
@@ -222,7 +222,7 @@ You can use different tokens to do it. Currently it works with USDC, WMATIC and
 ```typescript
 const weth = new ethers.Contract(wethAddress, wethAbi, signer);
 
-const offsetReceipt = await toucan.autoOffsetUsingSwapToken(
+const offsetReceipt = await toucan.autoOffsetUsingToken(
```

Comment:
the OffsetHelper has already been updated but the SDK was still based on the old versions. I changed the naming so it would fit the naming of the offsetHelper functions. But this is actually wrong update of the function name. Correct would be `autoOffsetExactInToken` . they had to be changed because before the function didn't differentiate between `autoOffsetExactInToken` and `autoOffsetExactOutToken`